From: Yiyang Chen <cyyzero16@gmail.com>
Subject: taskstats: remove dead taskstats_exit_mutex declaration
Date: Tue, 7 Jul 2026 22:52:05 +0800

The extern declaration of taskstats_exit_mutex has never been defined nor
referenced anywhere now.  Just remove it.

Link: https://lore.kernel.org/98948e69094b73d6dfa63dcf0770067b57f3becf.1783435695.git.cyyzero16@gmail.com
Signed-off-by: Yiyang Chen <cyyzero16@gmail.com>
Cc: Balbir Singh <balbirs@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/taskstats_kern.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/taskstats_kern.h~taskstats-remove-dead-taskstats_exit_mutex-declaration
+++ a/include/linux/taskstats_kern.h
@@ -14,7 +14,6 @@
 
 #ifdef CONFIG_TASKSTATS
 extern struct kmem_cache *taskstats_cache;
-extern struct mutex taskstats_exit_mutex;
 
 static inline void taskstats_tgid_free(struct signal_struct *sig)
 {
_
